src/main/java/frc/robot/commands/TrajectoryPresetSteerAngles.java

```diff
@@ -0,0 +1,62 @@
+package frc.robot.commands;
+
+import edu.wpi.first.math.geometry.Pose2d;
+import edu.wpi.first.math.geometry.Rotation2d;
+import edu.wpi.first.math.kinematics.ChassisSpeeds;
+import edu.wpi.first.math.kinematics.SwerveModuleState;
+import edu.wpi.first.math.trajectory.Trajectory;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.subsystems.Drivetrain;
+
+/**
+ * Sets all module angles to a given trajectory's initial angle.
+ */
+public class TrajectoryPresetSteerAngles extends CommandBase {
```

Comment:
Why would this class be useful?

Reply:
Used for preseting angles. we could use it befor match if you think it would slow down auto.